backend/prompt_studio/prompt_studio_registry_v2/tests/test_frame_export_json.py

```
@@ -0,0 +1,318 @@
+    _install(
+        "prompt_studio.prompt_studio_registry_v2.serializers",
+        {"PromptStudioRegistrySerializer": MagicMock(name="PromptStudioRegistrySerializer")},
+    )
+
+    from prompt_studio.prompt_studio_registry_v2 import (  # noqa: E402
+        prompt_studio_registry_helper as _psrh_mod,
+    )
+
+    PromptStudioRegistryHelper = _psrh_mod.PromptStudioRegistryHelper
+    _IMPORT_ERROR: str | None = None
+except Exception as exc:  # pragma: no cover — environment guard
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)